emr-cluster-broker.tf

```diff
@@ -0,0 +1,33 @@
+resource "github_repository" "emr-cluster-broker" {
+  name        = "emr-cluster-broker"
+  description = "A lightweight API to passthrough requests to AWS SDK and create EMR clusters"
+
+  allow_merge_commit = false
+  auto_init          = true

```

Comment:
you need to replace this line with `default_branch     = "master"` as the repo has already been created